### PR TITLE
Updated to use AppendAnsiLine

### PR DESCRIPTION
**Pull request by DrunkenImpGameStudio**
Updated to use AppendAnsiLine

### CODE REVIEW

**Comment by DavidRieman** [comment]
Hi, and welcome!  Definitely appreciate these!  I've only got a minute before I start my day job work day, but I'll be back to properly respond to things at day's end.

I merged in the new AppendAnsiLine - could you merge master into your PR branches and see if the automatic CI steps will pass again?  Thanks!

As for me - I am in the middle of trying to fix the prompt system (prompts were inconsistent and even the "> " wasn't showing up when it should, which revealed problems with the More paging system and word-wrapping system).  I also have some of the char creation steps updated for line endings locally; I'll make sure to resolve any merge conflicts on my end though.

Also, feel free to add yourself to the Credits.txt file as a contributor  :)

**Comment by DrunkenImpGameStudio** [comment]
Awesome. Looking forward to see this project progress. Merged, and now this ones passing. I'll recheck the other to see why its failing.

File: src/WarriorRogueMage/CharacterCreation/SetAttributesState.cs [hunk 1]
```diff
@@ -157,22 +157,22 @@ private void SetAttributeTarget(ref int targetPoints, int newValue)
...
-            sb.AppendLine("To add points to an attribute, use the + operator. Example: warrior +6");
-            sb.AppendLine("To subtract points from an attribute, use the - operator. Example: warrior -6");
-            sb.AppendLine("When you are done distributing the character points, type done.");
-            sb.AppendLine("====================================================================<%n%>");
+            sb.AppendAnsiLine();
+            sb.AppendAnsiLine();
+            sb.AppendAnsiLine("You have 10 character points to be divided between 3 attributes.");
+            sb.AppendAnsiLine("No attribute can have more than 6 points. Attributes can be zero.");
+            sb.AppendAnsiLine();
+            sb.AppendAnsiLine($"Warrior : {warriorPoints}" + AnsiSequences.NewLine);
```
Comment (DavidRieman) [inline src/WarriorRogueMage/CharacterCreation/SetAttributesState.cs:165]: nit: Could further improve these with the embedded newline `{warriorPoints}{AnsiSequences.NewLine}");` if you want.

**Review by DavidRieman — APPROVED** [review]
Thanks again!

**Pull request merged**